[18.0][MIG] mgmtsystem_review: Migration to 18.0

natuan9 opened the pull request:
Depend on:
- `https://github.com/OCA/management-system/pull/609`
- https://github.com/natuan9/management-system/pull/3

Commit:
[ADD] mgmtsystem_review

* Initial commit. Import from extra-addons.
* cleanup
* Group references. Reported by michaelgiesbrecht on the forum.
* Icons
* Move audittrail dependency from mgmtsystem to requiring modules

Commit:
Launchpad automatic translations update.

dreispt committed:
CHG: marking Apps, security review, translation pt, finetuning

Commit:
[ADD] Move wiki_quality_manual from lp:openobject-addons/6.1 to document_page_quality_manual

Commit:
[REN] wiki_* to document_page_*

dreispt committed:
[FIX+IMP] mgmgtsystem_review: Selections use words instead of letters; fixed AGPL-3 reference + .py files licenses to AGPL

Commit:
[FIX] Fix analysis after fixing the environments

Commit:
[FIX+MIG+IMP] mgmtsystem_review: 4 things

* [FIX] Bogus import in mgmtsystem_review.

  The wizard sub-module doesn't exist in this module and in openerp 7.0,
  it caused problems.

* [MIG] Added migration scripts for 0.1 --> 1.0 in the modules that needed it.

  mgmtsystem_nonconformity:

  * o --> open c --> done state change.
  * Migrating action relations to a many2many table.
  * Concatening action comments in a single field

  mgmtsystem_audit and mgmtsystem_review:

  * o --> open c --> done state change.

* [FIX] Fixed "Unknown field state for domain" errors.

  In 7.0, it's necessary, in a view, to have the field present in a view in order to use it
  in a domain. I added an invisible "state" field in relevant views to fix the errors.

* [IMP] Updated statusbar/buttons's layout to conform to OpenERP 7.0 style.

  Instead of a group at the bottom of the form, it's now a <header> tag at the top.

  Changed Actions, Audit, Nonconformity and Review.

Commit:
Launchpad automatic translations update.

Commit:
Launchpad automatic translations update.

Commit:
[CHG] Changed MediaWiki syntax in document pages to plain HTML.

document_page doesn't seem to support MediaWiki syntax anymore.

Commit:
[CHG] Changed MediaWiki syntax in document pages to plain HTML.

document_page doesn't seem to support MediaWiki syntax anymore.

dreispt committed:
[CHG] Making audit trail rules an option. They will be automatically installed if the audittrail module is installed.

Commit:
[FIX+IMP] mgmtsystem_review: Several things:

* [FIX] Migration folder names
* label Poplicy n'est pas au bonne endroit
* [UPG] module manifest and views to v7 format
* [IMP] Add company_id and its default value
* [IMP] Add company_id in views
* [FIX] Missing object : review_line
* [IMP] mgmtsystem_*_security.xml files
* [FIX] PEP8 compliance after running flake8

Commit:
Adding migrationscript for procedure for OpenUpgrade

Commit:
Adding migration script for quality manual for OpenUpgrade

dreispt committed:
Init 8.0 branch

llacroix committed:
[MIG] mgmtsystem_review: Migration to 8.0

Fix imports and replaced survey.responses by survey.user_input
Removed vim lines
[IMP] Add README.rst and data, views and models directories
Remove executable flag on python files and xml
Moved icon
[IMP] - mgmtsystem_review: Improved view and fixed typo.
[ADD] - mgmtsystem_review: Updated pot and added spanish translation.
[FIX] - mgmtsystem_review: Fixed translation template.
[REM] Migration files
[IMP] Description, version
[IMP] Add "Try me" button
[IMP] Remove osv
[FIX] Formatting in README.rst
[FIX] import models
[IMP] Migrate to new API
[FIX] AttributeError: 'module' object has no attribute 'Many2Many'
[FIX] Trailing comma after field definition

dreispt committed:
Init 8.0 branch

llacroix committed:
Moved document_page_procedure to root folder for port

Simple fix nothing to change

Refactor move icon and splitted xml file in data/views

llacroix committed:
Moved document_page_quality_manual to root folder for port

llacroix committed:
Making installable is enough, nothing to port

Commit:
[IMP] Add README.rst and data directory

Commit:
[FIX] No reference to document_page_group_procedure

llacroix committed:
Moved icon and reformatted manifest

llacroix committed:
Review changes from PR#35

Removed certificate from __openerp__

Remove explicit view ids in action window

sysadminmatmoz committed:
Slovene translations added

yvaucher committed:
Add bug tracker link on README.rst

Commit:
[ADD] - document_page_procedure: Added assigned fixed views, updated pot and added spanish translations.

oca-transbot committed:
OCA Transbot updated translations from Transifex

sbidoul committed:
[UPD] prefix versions with 8.0

pedrobaeza committed:
[MIG] Make modules uninstallable

pedrobaeza committed:
[MIG] Make modules uninstallable

naousse committed:
Migration of module mgmtsystem_manual from Odoo 8.0 to Odoo 9.0

naousse committed:
Migration of document_page_procedure to Odoo 9.0

naousse committed:
Migration of document_page_quality_manual to Odoo 9.0

[IMP] Update the structure to ISO 9001:2015

Commit:
[IMP] Add menu Categories

Update README.rst

[IMP] Add menu Categories

[IMP] Menu sequence

[FIX] Manuals action domain

oca-transbot committed:
OCA Transbot updated translations from Transifex

Commit:
Update README.rst

naousse committed:
[MIG] mgmtsystem_review: Migration to 9.0

sbidoul committed:
[FIX] remove en.po that was erroneously created by transbot

sbidoul committed:
[FIX] remove en.po that was erroneously created by transbot

oca-transbot committed:
OCA Transbot updated translations from Transifex

pedrobaeza committed:
[MIG] Make modules uninstallable

pedrobaeza committed:
[MIG] Make modules uninstallable

pedrobaeza committed:
[MIG] Make modules uninstallable

pedrobaeza committed:
[MIG] Rename manifest files

pedrobaeza committed:
[MIG] Rename manifest files

pedrobaeza committed:
[MIG] Rename manifest files

eugen-don committed:
MIG-mgmtsystem_manuals-to-V.-10.0

MIG-mgmtsystem_manuals-to-V.-10.0

eugen-don committed:
MIG document_page_procedure to V. 10.0

MIG document_page_procedure to V. 10.0

eugen-don committed:
MIG document_page_quality_manual to V. 10.0

MIG document_page_quality_manual to V. 10.0

ThomasBinsfeld committed:
[IMP] mgmtsystem: document_page_approval as option

oca-transbot committed:
OCA Transbot updated translations from Transifex

eugen-don committed:
MIG mgmtsystem_review to V. 10.0

benwillig committed:
[FIX] Added specific tree view for res users with create=1 and delete=1 to allow users selection on review

jalzaga committed:
[MIG] document_page_procedure: Migration to 11.0

jalzaga committed:
[MIG] mgmtsystem_manual: Migration to 11.0

jalzaga committed:
[MIG] document_page_quality_manual: Migration to 11.0

jalzaga committed:
[MIG] mgmtsystem_review: Migration to 11.0

oca-transbot committed:
Update translation files

Updated by Update PO files to match POT (msgmerge) hook in Weblate.

oca-transbot committed:
Update translation files

Updated by Update PO files to match POT (msgmerge) hook in Weblate.

primes2h committed:
Added translation using Weblate (Italian)

primes2h committed:
Added translation using Weblate (Italian)

Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: management-system-11.0/management-system-11.0-mgmtsystem_manual
Translate-URL: https://translation.odoo-community.org/projects/management-system-11-0/management-system-11-0-mgmtsystem_manual/it/

gurneyalex committed:
[MIG] migration to 12.0

Commit:
[MIG] document_page_procedure: migrate to v12

jcoux committed:
[MIG] 12.0 - mgmtsystem_review

Commit:
[MIG] document_page_quality_manual: migrate to v12

sbidoul committed:
[FIX] bad version number on 12.0 branch

ernestotejeda committed:
[FIX] Fixing pylint errors

Modules: document_page_quality_manual, document_page_work_instruction,
mgmtsystem_action

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (53 of 53 strings)

Translation: management-system-12.0/management-system-12.0-mgmtsystem_review
Translate-URL: https://translation.odoo-community.org/projects/management-system-12-0/management-system-12-0-mgmtsystem_review/pt_BR/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (5 of 5 strings)

Translation: management-system-12.0/management-system-12.0-mgmtsystem_manual
Translate-URL: https://translation.odoo-community.org/projects/management-system-12-0/management-system-12-0-mgmtsystem_manual/pt_BR/

awesome-manuel committed:
Translated using Weblate (German)

Currently translated at 100.0% (53 of 53 strings)

Translation: management-system-12.0/management-system-12.0-mgmtsystem_review
Translate-URL: https://translation.odoo-community.org/projects/management-system-12-0/management-system-12-0-mgmtsystem_review/de/

awesome-manuel committed:
Translated using Weblate (German)

Currently translated at 100.0% (5 of 5 strings)

Translation: management-system-12.0/management-system-12.0-mgmtsystem_manual
Translate-URL: https://translation.odoo-community.org/projects/management-system-12-0/management-system-12-0-mgmtsystem_manual/de/

ernestotejeda committed:
[IMP] mgmtsystem_manual: black, isort

ernestotejeda committed:
[MIG] mgmtsystem_manual: Migration to 13.0

ernestotejeda committed:
[IMP] document_page_procedure: black, isort

ernestotejeda committed:
[MIG] document_page_procedure: Migration to 13.0

ernestotejeda committed:
[IMP] document_page_quality_manual: black, isort

ernestotejeda committed:
[MIG] document_page_quality_manual: Migration to 13.0

ernestotejeda committed:
[IMP] mgmtsystem_review: black, isort

ernestotejeda committed:
[MIG] mgmtsystem_review: Migration to 13.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: management-system-13.0/management-system-13.0-mgmtsystem_review
Translate-URL: https://translation.odoo-community.org/projects/management-system-13-0/management-system-13-0-mgmtsystem_review/

Commit:
Multicompany style

Rad0van committed:
Changes to common display of forms

dzungtran89 committed:
[IMP] document_page_quality_manual: black, isort, prettier

dzungtran89 committed:
[MIG] document_page_quality_manual: Migration to 14.0

dzungtran89 committed:
[IMP] document_page_procedure: black, isort, prettier

dzungtran89 committed:
[MIG] document_page_procedure: Migration to 14.0

dzungtran89 committed:
[IMP] mgmtsystem_manual: black, isort, prettier

dzungtran89 committed:
[MIG] mgmtsystem_manual: Migration to 14.0

dzungtran89 committed:
[IMP] mgmtsystem_review: black, isort, prettier

dzungtran89 committed:
[MIG] mgmtsystem_review: Migration to 14.0

Commit:
[ADD] Add chatter to mgmtsystem.review

victoralmau committed:
[FIX] mgmtsystem_manual: Set correct domain and context in action manual (auto-fill manual category).
TT32895

OCA-git-bot committed:
mgmtsystem_manual 14.0.1.1.0

YvanDotet committed:
[MIG] document_page_quality_manual: Migration to 15.0

[UPD] noupdate data

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 60.2% (53 of 88 strings)

Translation: management-system-14.0/management-system-14.0-mgmtsystem_review
Translate-URL: https://translation.odoo-community.org/projects/management-system-14-0/management-system-14-0-mgmtsystem_review/it/

YvanDotet committed:
[MIG] document_page_procedure: Migration to 15.0

YvanDotet committed:
[MIG] mgmtsystem_manuel: Migration to 15.0

ivantodorovich committed:
[UPD] Modules development status : solve dev status dependency check

YvanDotet committed:
[MIG] mgmtsystem_review: Migration to 15.0

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: management-system-15.0/management-system-15.0-mgmtsystem_manual
Translate-URL: https://translation.odoo-community.org/projects/management-system-15-0/management-system-15-0-mgmtsystem_manual/it/

FernandoRomera committed:
[16.0][MIG] mgmtsystem_manual: Migration to 16.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: management-system-16.0/management-system-16.0-mgmtsystem_manual
Translate-URL: https://translation.odoo-community.org/projects/management-system-16-0/management-system-16-0-mgmtsystem_manual/

FernandoRomera committed:
[16.0][MIG] document_page_procedure: Migration to 16.0

FernandoRomera committed:
[16.0][MIG] document_page_quality_manual: Migration to 16.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: management-system-16.0/management-system-16.0-document_page_quality_manual
Translate-URL: https://translation.odoo-community.org/projects/management-system-16-0/management-system-16-0-document_page_quality_manual/

FernandoRomera committed:
[16.0][MIG] mgmtsystem_review: Migration to 16.0

Commit:
[MIG] mgmtsystem_review: Migration to 17.0

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (5 of 5 strings)

Translation: management-system-16.0/management-system-16.0-mgmtsystem_manual
Translate-URL: https://translation.odoo-community.org/projects/management-system-16-0/management-system-16-0-mgmtsystem_manual/es/

m1k3lm committed:
[IMP] document_page_procedure: pre-commit auto fixes

m1k3lm committed:
[MIG] document_page_procedure: Migration to 17.0

Commit:
[IMP] mgmtsystem_manual: pre-commit auto fixes

Commit:
[MIG] mgmtsystem_manual: Migration to 17.0

mostafabarmshory committed:
Added translation using Weblate (Persian)

mostafabarmshory committed:
Added translation using Weblate (Persian)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: management-system-17.0/management-system-17.0-mgmtsystem_manual
Translate-URL: https://translation.odoo-community.org/projects/management-system-17-0/management-system-17-0-mgmtsystem_manual/

natuan9 committed:
[IMP] document_page_procedure: pre-commit auto fixes

natuan9 committed:
[MIG] document_page_procedure: Migration to 18.0

natuan9 committed:
[IMP] mgmtsystem_manual: pre-commit auto fixes

natuan9 committed:
[MIG] mgmtsystem_manual: Migration to 18.0

Kimkhoi3010 committed:
[IMP] document_page_quality_manual: pre-commit auto fixes

Kimkhoi3010 committed:
[MIG] document_page_quality_manual: Migration to 18.0

natuan9 committed:
add checklog-odoo.cfg

natuan9 committed:
[IMP] mgmtsystem_review: pre-commit auto fixes

natuan9 committed:
[MIG] mgmtsystem_review: Migration to 18.0

Kimkhoi3010 commented:
Please squash administrative commits (if any) with the previous commit for reducing commit noise. Check https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate for details.

Kimkhoi3010 commented on mgmtsystem_review/readme/CREDITS.md:
```diff
@@ -0,0 +1,3 @@
+The development of this module has been financially supported by:
+
+- Camptocamp
```
Please also update file readme/CREDITS.md:
The migration of this module from 17.0 to 18.0 was financially supported by Camptocamp.

Kimkhoi3010 commented on mgmtsystem_review/readme/CONTRIBUTORS.md:
```diff
@@ -0,0 +1,15 @@
+Trobz
+
+- Dung Tran \<<dungtd@trobz.com>\>
+- Yvan Dotet \<<yvan.dotet@logicasoft.eu>\>
```
Yvan Dotet is not part of Trobz, we should move him out of Trobz's contributor block.
btw, on this occasion let's revise our Trobz block to the same format as the other companies above.
example: - [Trobz](https://www.trobz.com):

Kimkhoi3010 reviewed (approved):
LGTM

natuan9 committed:
[DON'T MERGE] test-requirements.txt

OCA-git-bot committed:
Merge PR #610 into 18.0

Signed-off-by max3903

Commit:
[UPD] Update document_page_procedure.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: management-system-18.0/management-system-18.0-document_page_procedure
Translate-URL: https://translation.odoo-community.org/projects/management-system-18-0/management-system-18-0-document_page_procedure/

OCA-git-bot committed:
Merge PR #606 into 18.0

Signed-off-by max3903

Commit:
[UPD] Update mgmtsystem_manual.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #608 into 18.0

Signed-off-by max3903

Commit:
[UPD] Update document_page_quality_manual.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: management-system-18.0/management-system-18.0-document_page_quality_manual
Translate-URL: https://translation.odoo-community.org/projects/management-system-18-0/management-system-18-0-document_page_quality_manual/
